lib/utilities/paginator.rb

```diff
@@ -0,0 +1,106 @@
+module Utilities
+  module Paginator
+    def items_to_display(model:, items_per_page: 5, current_page: 1)
+      @items_per_page = items_per_page
+      @current_page = current_page.to_i
```

Comment:
Why are we calling `to_i` on current page? I would assume we are passing in a integer to this method and not a float or string.